src/composables/multisig.ts

```diff
@@ -71,13 +112,182 @@ export function useMultisig({ store }: UseMultisigOptions) {
     multisigProgress.value = MULTISIG_CREATION_STEPS.created;
 
     return {
-      gaContractId: gaContract.contractId,
+      contractId: gaContract.contractId,
       gaAccountId: gaAccount.publicKey,
     };
   }
 
+  async function getTransactionByHash(txHash: string) {
+    return fetchJson(`${activeNetwork.value.multisigBackend}/tx/${txHash}`);
+  }
+
+  function setActiveGaAccountId(gaAccountId: string) {
+    activeGaAccountId.value = gaAccountId;
+  }
+
+  async function updateMultisigAccounts() {
+    const sdk = await getSdk();
+
+    let rawMultisigData: IMultisigAccount[] = [];
+
+    try {
+      await Promise.all(accounts.value.map(async ({ address }) => rawMultisigData.push(
+        ...(await fetchJson(`${activeNetwork.value.multisigBackend}/${address}`)),
+      )));
+    } catch {
+      // eslint-disable-next-line no-console
+      console.log('failed to fetch multisigAccounts');
```

Comment:
If this is a temporary solution please ass `TODO` with some comment. otherwise I suggest to properly handle the error.